src/perftest_resources.c

```diff
@@ -1618,9 +1598,9 @@ int create_reg_qp_main(struct pingpong_context *ctx,
 	}
 
 	if (ctx->qp[i] == NULL) {
-		fprintf(stderr, "Unable to create QP.\n");
-		return FAILURE;
-	}
+			return FAILURE;
```

Comment:
Same question, is this related?

Reply:
> Same question, is this related?

Yes, it is, because we don't want to print this in fall-back case